core/db/migrate/20151010003252_add_foreign_keys_for_taxons_prototypes.rb

```diff
@@ -0,0 +1,7 @@
+class AddForeignKeysForTaxonsPrototypes < ActiveRecord::Migration
+  def change
+    { spree_taxons: :taxon_id, spree_prototypes: :prototype_id }.each do |to_table, column|
+      add_foreign_key :spree_prototype_taxons, to_table, column: column, on_delete: :cascade
```

Comment:
I'm pro referential integrity. My only question is on the cascade.  There might be some benefits to having the cascades be done in activerecord?  I'm not sure.